src/Frontend/Modules/Blog/Actions/Index.php

```diff
@@ -29,6 +29,10 @@ private function buildPaginationConfig(): array
         $limit = $this->get('fork.settings')->get($this->getModule(), 'overview_num_items', 10);
         $numberOfPages = (int) ceil($numberOfItems / $limit);
 
+        if ($numberOfPages === 0) {
+            $numberOfPages = 1;
+        }
```

Comment:
Changing the exact same thing in two locations might indicate that we're missing a concept in our data model here. A pagination object/trait might be a nice cleanup?

Reply:
This is just a small method to build the config. the biggest part of the pagination is already extracted